### PR TITLE
- Tabbar integrated to main page

### PR DESCRIPTION
**Pull request by kataparuto**
-The related Fragment, layout and Adapter are created
- MainActivity layout created
- After login, MainActivity is displayed instead of ProfileActivity now
- HomeFragment is created as the first fragment of the app

### CODE REVIEW

File: android/AndroidSportsPlatform/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
         <activity android:name=".activities.SearchOperationsActivity" />
         <activity android:name=".activities.RegisterActivity" />
         <activity android:name=".activities.ProfileActivity" />
+        <activity android:name=".activities.MainActivity" />
```
Comment (omersuve) [inline android/AndroidSportsPlatform/app/src/main/AndroidManifest.xml:21]: Changing name _Main_ to _Login_ fits better for login operation.

File: android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/activities/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    override val kodein by kodein()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val binding: ActivityMainBinding = DataBindingUtil.setContentView(
+            this, R.layout.activity_main)
+        val view = binding.root
+        setContentView(view)
+
```
Comment (omersuve) [inline android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/activities/MainActivity.kt:22]: An Independent Main Activity is a good approach to make tasks more seperated.

File: android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/adapter/TabBarPagerAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        }
+    }
+
+    override fun getPageTitle(position: Int): CharSequence {
+        return when (position) {
+            0 -> context.resources.getString(R.string.tabBarHome)
+            1 -> context.resources.getString(R.string.tabBarSearch)
+            2 -> context.resources.getString(R.string.tabBarProfile)
+            else -> ""
+        }
```
Comment (omersuve) [inline android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/adapter/TabBarPagerAdapter.kt:35]: We may create a Unit Test for these functions to know whether they are always working well.

File: android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/fragments/TabBarFragment.kt [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        super.onDestroyView()
+        _binding = null
+    }
+
+    private fun initializeTabBarPager() {
+        binding.viewPagerTabBar.apply {
+            offscreenPageLimit = TABBAR_COUNT
+            adapter = tabBarAdapter
+        }
+    }
```
Comment (omersuve) [inline android/AndroidSportsPlatform/app/src/main/java/com/example/sportsplatform/fragments/TabBarFragment.kt:52]: We may implement a Unit test for this function.

File: android/AndroidSportsPlatform/app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (omersuve) [inline android/AndroidSportsPlatform/app/src/main/res/layout/fragment_home.xml:17]: Later, we should add the attended events of the logged in User in this fragment layout.

**Review by omersuve — APPROVED** [review]
As we decided; using fragments seems useful for later implementations, and a tabbar that constantly exists is practical for users.

File: android/AndroidSportsPlatform/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (kataparuto) [reply android/AndroidSportsPlatform/app/src/main/AndroidManifest.xml:21]: Exactly.

**Pull request merged**